tests/bluetooth/tester/src/btp/btp_has.h

```diff
@@ -0,0 +1,61 @@
+/* btp_has.h - Bluetooth tester headers */
+
+/*
+ * Copyright (c) 2023 Oticon
```

Comment:
Just curious question: Should this be `Demant` now, or are you guys sticking with `Oticon` for copyrights?